### DISCUSSION TITLE
Release of bugfix version 0.7.5

### DISCUSSION THREAD

**Discussion by lgblaumeiser** [eclipse-tractusx/tractusx-edc#1542] [General]
As we currently work on a bugfix 0.7.5 that fixes additional issues found in 0.7.4, let us track here the concrete issues that are incorporated in this bugfix release version:

**Comment by rafaelmag110** [comment]
This PR that adds missing AzureStorage provisioning extensions. https://github.com/eclipse-tractusx/tractusx-edc/pull/1537

**Comment by rafaelmag110** [comment]
This PR that backports a fix to a memory-leak in the upstream edc v0.7.2 implementation of the PipelineServiceImpl class: https://github.com/eclipse-tractusx/tractusx-edc/pull/1520

**Comment by rafaelmag110** [comment]
This PR that backports a fix for correct termination of non existing dataflows: https://github.com/eclipse-tractusx/tractusx-edc/pull/1534

**Reply by rafaelmag110** [reply]
This might lead existing TPs to stay in an infinite STARTED state.
